tests/integration/test_interaction.py

```diff
@@ -305,7 +354,7 @@ def _get_scenario_xml(self):  # pylint: disable=no-self-use
         return "<vertical_demo><drag-and-drop-v2/></vertical_demo>"
 
 
-class BasicInteractionTest(DefaultDataTestMixin, InteractionTestBase):
+class BasicInteractionTest(DefaultDataTestMixin, InteractionTestBase, BaseIntegrationTest):
```

Comment:
@itsjeyd Ah good catch. I was trying to run `BasicInteractionTest` in isolation, but it would fail with errors such as "missing setUp() method". I later realized that this class was not supposed to be run in isolation and was just a mixin that would have its `test_*` methods run when mixed into other tests such as `KeyboardInteractionTest`.
Having mouse manipulation tests run inside `KeyboardInteractionTest` was weird, so I in the most recent commit I renamed `BasicInteractionTest` to `StandardInteractionTest` and made it an actual test class that can be run in isolation. I also made both mouse and keyboard intractions.